strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java

```diff
@@ -227,15 +232,37 @@ private FileInputStream extractPackageMultipartStream(String boundaryString,
     }
 
     private void writePackagePart(String boundaryString,
-                                  ServletInputStream is,
+                                  InputStream is,
                                   FileOutputStream packagePartOutputStream)
-            throws IOException
+        throws IOException
     {
+        //We need to fix missing starting byte of ending mulipart boundary (0x0D), which is incorrectly generated by NuGet `push` implementation
```

Comment:
@carlspring you need to check this out :)
I spent a lot of time to fix this tricky issue with NuGet push multipart.